test/e2e_test.go

```diff
@@ -114,162 +106,109 @@ func createTestSuite(t *testing.T, useMemory bool) (TestSuite, func()) {
 	}, kill
 }
 
-func TestHoleskyWithPlasmaClient(t *testing.T) {
-	if !runTestnetIntegrationTests {
-		t.Skip("Skipping testnet integration test")
```

Comment:
Can we please revert these testing changes? Made the same adjustments and added parallelisation in the OP testing PR

Reply:
Done